ce_main_app/ikbd/ikbd_general.cpp

```diff
@@ -225,7 +227,7 @@ Ikbd::Ikbd()
 void Ikbd::loadSettings(void)
 {
     Settings s;
-    bool firstJoyIs0 = s.getBool((char *) "JOY_FIRST_IS_0", false);
+    firstJoyIs0 = s.getBool((char *) "JOY_FIRST_IS_0", false);
```

Comment:
No idea, as you can see, it was there before, I just made it a member variable...